BUILD.md

```diff
@@ -32,7 +32,7 @@ conda remove --force rapids-build-env rapids-notebook-env rapids-doc-env
 
 ### Recommended process
 
-As a convenience, a `build.sh` script is provided which can be used to execute the same build commands above.  Note that the libraries will be installed to the location set in `$INSTALL_PREFIX` if set (i.e. `export INSTALL_PREFIX=/install/path`), otherwise to `$CONDA_PREFIX`.
+As a convenience, a `build.sh` script is provided which can be used to execute the same build commands as below.  Note that the libraries will be installed to the location set in `$INSTALL_PREFIX` if set (i.e. `export INSTALL_PREFIX=/install/path`), otherwise to `$CONDA_PREFIX`.
```

Comment:
```suggestion
As a convenience, a `build.sh` script is provided which can be used to execute the same build commands in the [manual process](#manual-process) section below.  Note that the libraries will be installed to the location set in `$INSTALL_PREFIX` if set (i.e. `export INSTALL_PREFIX=/install/path`), otherwise to `$CONDA_PREFIX`.
```